crates/protect-ffi/src/lib.rs

```diff
@@ -817,18 +523,17 @@ fn format_index_term_ore(bytes: &Vec<u8>) -> Vec<String> {
     vec![format_index_term_ore_bytea(bytes)]
 }
 
-fn is_defined(js_value: Handle<'_, JsValue>, cx: &mut FunctionContext) -> bool {
-    !js_value.is_a::<JsUndefined, _>(cx)
-}
+static RUNTIME: OnceCell<Runtime> = OnceCell::new();
 
 #[neon::main]
 fn main(mut cx: ModuleContext) -> NeonResult<()> {
-    cx.export_function("newClient", new_client)?;
-    cx.export_function("encrypt", encrypt)?;
-    cx.export_function("encryptBulk", encrypt_bulk)?;
-    cx.export_function("decrypt", decrypt)?;
-    cx.export_function("decryptBulk", decrypt_bulk)?;
-    cx.export_function("decryptBulkFallible", decrypt_bulk_fallible)?;
+    let runtime = RUNTIME
+        .get_or_try_init(Runtime::new)
+        .or_else(|err| cx.throw_error(err.to_string()))?;
+
+    let _ = neon::set_global_executor(&mut cx, runtime);
+
+    neon::registered().export(&mut cx)?;
```

Comment:
This isn't documented well in the Neon docs, but we need to call `neon::registered().export(&mut cx)` ourselves here since we define our own `neon::main` function.

We need to define our own `neon::main` function to set up the Tokio runtime.